src/components/AccountDropdownComponent.jsx

```diff
@@ -0,0 +1,176 @@
+import InputLabel from '@material-ui/core/InputLabel'
+import MenuItem from '@material-ui/core/MenuItem'
+import FormControl from '@material-ui/core/FormControl'
+import OutlinedInput from '@material-ui/core/OutlinedInput'
+import Divider from '@material-ui/core/Divider'
+import Typography from '@material-ui/core/Typography'
+import Skeleton from '@material-ui/lab/Skeleton'
+import { accountStatus } from '../types/account.flow'
+import { getCryptoSymbol, getCryptoLogo } from '../tokens.js'
+import { getWalletTitle } from '../wallet.js'
+
+type Props = {
+  classes: Object,
+  account: ?Object,
+  cryptoAccounts: Array<Object>,
+  error: Object,
+  onChange: Function,
+  toCurrencyAmount: Function
+}
+
+class AccountDropdownComponent extends Component<Props> {
+  renderAccountItem = item => {
+    const { classes, toCurrencyAmount } = this.props
+
+    if (item.skeletonOnly) {
+      return (
+        <React.Fragment>
+          <Box pr={1} mb={1}>
+            <Skeleton variant='circle' width={40} height={40} />
+          </Box>
+          <Skeleton height={6} />
+          <Skeleton height={6} width='80%' />
+        </React.Fragment>
+      )
+    }
+
+    return (
+      <Grid container direction='row' alignItems='center'>
+        {/* crypto icon */}
+        <Grid item xs={1}>
+          <Avatar src={getCryptoLogo(item.cryptoType)}></Avatar>
+        </Grid>
+        {/* name and address */}
+        <Grid item xs={8}>
+          <Grid container direction='column'>
+            <Grid item>
+              <Typography variant='body2'>
+                {item.name} ({getWalletTitle(item.walletType)})
+              </Typography>
+            </Grid>
+            <Grid item>
+              <Typography variant='caption'>{item.address}</Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+        {/* balance */}
+        <Grid item xs={3}>
+          <Grid container direction='column' alignItems='flex-end'>
+            <Grid item>
+              <Typography variant='body2'>
+                {item.status === accountStatus.syncing ? (
+                  <Skeleton style={{ margin: '0px', width: '100%', minWidth: '100px' }} />
+                ) : (
+                  `${item.balanceInStandardUnit} ${getCryptoSymbol(item.cryptoType)}`
+                )}
+              </Typography>
+            </Grid>
+            <Grid item>
+              <Typography variant='caption'>
+                {toCurrencyAmount(item.balanceInStandardUnit, item.cryptoType)}
+              </Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+      </Grid>
+    )
+  }
+
+  render () {
+    const { classes, account, cryptoAccounts, onChange, pending, error } = this.props
+    let skeletonCryptoAccounts = []
+    if (pending) {
+      skeletonCryptoAccounts = [
+        { skeletonOnly: true },
+        { skeletonOnly: true },
+        { skeletonOnly: true }
+      ]
+    }
+
+    return (
+      <>
+        <Grid item>
```

Comment:
it is bad to use <Grid item> as the wrapper of a component.
Use <Grid container> or something similar.
In this case, try wrap <FormControl> and <Box> inside a <Grid container> and combine with suggested change in TransferFormComponent